web_external/templates/body/frontPage.jade

```diff
@@ -1,56 +1,102 @@
-.isic-body-nopad
-  .isic-frontpage-container
-    .isic-header
-      .isic-site-title ISIC 
-      .isic-site-desc International Skin Imaging Collaboration
-    .isic-banner-menu
-      .isic-banner-menu-buttons
-        ul
-          li About ISIC
-          li Upload Dataset
-          li Annotation Studies
-          li Feature Sets
-    .isic-body-items
-      .isic-gallery
-        .isic-section-text Gallery of Current Images
-        iframe.isic-frame(src='#{staticRoot}/built/plugins/isic_archive/extra/img/lesiondatasets.png')
-        .isic-datasets
-          .isic-section-text Current Datasets
-          .isic-datasets-body
-            .isic-datasets-bodyh2 ISIC_UDA-1:
-            .isic-datasets-bodyp
-              | Twelve pigmented lesion clinics from Europe, the United States and Australia contributed lesions to the is data set. The data set includes 557 cutaneous melanoctic lesions from adults (>=18 years of age) with either a definitive histopathologic diagnosis (for excised lesions), or a documented history of being clinically benign through review of serial clinical images acquired at least 6 months apart. Lesions were excluded from the data set if there was equivocal histopathology, unacceptable image quality, or if the lesions were located on the face, acral or mucosal sites.
-            .isic-datasets-bodyh2 ISIC_UDA-2:
-            .isic-datasets-bodyp
-              | This dataset contains 44 high quality lesion images of melanomas and benign lesions. Along with the image data, metadata is available related to patient age, diagnosis, gender, and anatomic location. For this dataset, 4 expert dermoscopists independently reviewed and annotated all the lesions images for dermoscopic patterns and structures.
-            .isic-datasets-bodyh2 ISIC_MSK-1:
-            .isic-datasets-bodyp
-              | The lesions included in this data set were acquired from a single pigmented lesion clinic. This data set only includes lesions that were excised. Relevant clinical metadata such as diagnosis, anatomic location, gender age and clinical impression of the lesion are included with each lesion image.
-            .isic-datasets-bodyh2 ISIC_MSK-2:
-            .isic-datasets-bodyp
-              | Benign and malignant skin lesions. Biopsy-confirmed melanocytic and non-melanocytic lesions.
-            .isic-datasets-bodyh2 ISIC_SONIC-1:
-            .isic-datasets-bodyp
-              | Moles in children. Benign melanocytic lesions from a pediatric population. The benign nature of the lesions is based on clinical assessment and/or no change on serial imaging.
-      .isic-intro
-        .isic-section-text Introduction
-        .isic-intro-body
+.isic-body-nopad.isic-frontpage
+  .isic-frontpage-header
+    .container
+      h1 ISIC Archive
+      h2 International Skin Imaging Collaboration: Melanoma Project
+  .isic-frontpage-body.container
+    .row
+      .col-lg-6
+        h3 Introduction
+        p
```

Comment:
It could be cleaner to write these paragraphs using `p.` instead of prefixing each line with `|`.
See http://jade-lang.com/reference/plain-text/, ("Block in a Tag")